proxy.php

```diff
@@ -11,9 +11,25 @@
  * @license  GPL http://www.gnu.org/licenses/gpl.html
  * @link     https://rudloff.pro/
  */
+$path =  realpath(dirname(__FILE__));
```

Comment:
No, because the proxy.php file not refer to WordPress functions. Functions available in WordPress will work only in the main file (gravatar-proxy.php)